PowerValidatedSolutions.psm1

```diff
@@ -32103,6 +32103,277 @@ Function Undo-NsxtRemoteTunnelEndpoint {
+        This example configures the NSX Global Manager to standby mode.
+        
+        .PARAMETER displayName
+        Display name to be assinged to the standby NSX Global Manager.
+
+        .PARAMETER standbyServer
+        The fully qualified domain name of the standby NSX Global Manager.
+
+        .PARAMETER standbyServerUser
+        The username to authenticate to the standby NSX Global Manager.
+
+        .PARAMETER standbyServerPass
+        The password to authenticate to the standby NSX Global Manager.
+    #>
+    
+    Param (
+        [Parameter (Mandatory = $true)] [ValidateNotNullOrEmpty()] [String]$displayName,  
+        [Parameter (Mandatory = $true)] [ValidateNotNullOrEmpty()] [String]$standbyServer,
+        [Parameter (Mandatory = $true)] [ValidateNotNullOrEmpty()] [String]$standbyServerUser,
+        [Parameter (Mandatory = $true)] [ValidateNotNullOrEmpty()] [String]$standbyServerPass
+    )
+
+    Try {
+        if ($nsxtHeaders.Authorization) {
+                #Custom Object to create json body for Connection Info
+                $gmnodethumbprint = (Get-SHA256Thumbprint -url "https://$standbyServer").replace(":", "")
+                $connection_info = New-Object -TypeName psobject
+                $connection_info | Add-Member -Notepropertyname 'fqdn' -Notepropertyvalue $standbyServer
+                $connection_info | Add-Member -Notepropertyname 'username' -Notepropertyvalue $user
+                $connection_info | Add-Member -Notepropertyname 'password' -Notepropertyvalue $pass
+                $connection_info | Add-Member -Notepropertyname 'thumbprint' -Notepropertyvalue $gmnodethumbprint
+                $conn_info = @()
+                $conn_info += $connection_info
+                $body = New-Object -TypeName psobject
+                $body | Add-Member -Notepropertyname 'display_name' -Notepropertyvalue $displayName
+                $body | Add-Member -Notepropertyname 'connection_info' -Notepropertyvalue $conn_info
+                $body | Add-Member -Notepropertyname 'mode' -Notepropertyvalue "STANDBY"
+                $body = $body | ConvertTo-Json -Depth 5
+                $uri = "https://$nsxtManager/global-manager/api/v1/global-infra/global-managers/$displayName"
+                $response = Invoke-RestMethod -Uri $uri -Method PUT -Headers $Global:nsxtHeaders -body $body -SkipCertificateCheck 
+            # Check the result of setting the secondary Global Manager as standby
+            if ($response) { 
+                Write-Output "Configuring the $nsxtManager NSX Global Manager as STANDBY : SUCCESSFUL"
+            }
+        }
+        else {
+            Write-Error "Not connected to NSX Local/Global Manager, run Request-NsxtToken and try again"
+        }
+    } Catch {
+        Write-Error $_.Exception.Message
+    }
+}
+Export-ModuleMember -Function Set-NsxtGloblaManagerStandby 
+
+Function Test-NsxVersionCompatibility {
```

Comment:
This same function can be used to check the compatability for standby and Active. Change inputs to fqdn, username and password